tests/helper/helper_generic.go

```diff
@@ -311,3 +312,14 @@ func CommonAfterEach(commonVar CommonVar) {
 
 	os.Unsetenv("GLOBALODOCONFIG")
 }
+
+// GenFileURL Converts file path on windows to /C:/path/to/file to work in URL
+func GenFileURL() string {
+	var sourcePath string
+	if runtime.GOOS == "windows" {
+		sourcePath = "file:///./"
+		return sourcePath
+	}
+	sourcePath = "file://./"
```

Comment:
this is not the correct local URI see https://en.wikipedia.org/wiki/File_URI_scheme#How_many_slashes?

Reply:
> this is not the correct local URI see https://en.wikipedia.org/wiki/File_URI_scheme#How_many_slashes?

@kadel Yes, agreed. I have analysed this thoroughly and commented out all of my observation here https://github.com/openshift/odo/issues/3350#issuecomment-735639702 . Also I have modified the PR accordingly, so cancelling the hold.

/hold cancel